.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,24 @@
+name: Integration Test
+
+on: [push]
```

Comment:
@RonWang 
1) It typically finishes before all unit tests (since there 9 variations of os/node version) - so I think it is ok to keep on every push. It is free.

2) We locked ask-smapi-model so I dont think there is point to run on cron. What benefit would it give us?

Reply:
1) It typically finishes before all unit tests (since there 9 variations of os/node version) - so I think it is ok to keep on every push. It is free.

2) We locked ask-smapi-model so I dont think there is point to run on cron. What benefit would it give us?

Reply:
cool. unit test and integration test will be triggered once feature branch push event happens.